.github/workflows/ci.yml

```diff
@@ -19,3 +19,8 @@ jobs:
       run: flake8 --show-source --statistics
     - name: Run tests
       run: pytest
+  reformat:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: psf/black@stable
```

Comment:
> Please note that the [--check flag](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#labels-exit-code) is required so that the workflow fails if Black finds files that need to be formatted.
`options: "--check --verbose"`

If we want to fail code contributions that do not follow code standard we need to add 
Ref: https://black.readthedocs.io/en/stable/integrations/github_actions.html

Reply:
i see that the --check is default, no need to add it ... :+1: